### PR TITLE
Update met.markdown

### LINKED ISSUE #9759: met.no documentation needs to be updated after breaking change
Please update the documentation for met.no

see https://github.com/home-assistant/home-assistant/issues/24824
THX

### PR DESCRIPTION
**Pull request by aidbish**
Update met.no to show config via configuration.yaml is deprecated and is to be added via UI

closes #9759

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9765"></a>

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please rebase it to `current`

File: source/_components/met.markdown [hunk 1]
```diff
@@ -18,29 +18,31 @@ redirect_from:
 
 The `met` platform uses the [Met.no](https://met.no/) web service as a source for meteorological data for your location. The weather forecast is delivered by the Norwegian Meteorological Institute and the NRK.
 
+NOTE: Configuration of the Met.no platform via configuration.yaml is deprecated from version 0.95
```
Comment (klaasnicolaas) [inline source/_components/met.markdown]: Put it in this [kind of note](https://developers.home-assistant.io/docs/en/documentation_create_page.html#html)

File: source/_components/met.markdown [hunk 2]
```diff
@@ -18,29 +18,31 @@ redirect_from:
...
+
 ## {% linkable_title Configuration %}
 
-To add Met to your installation, add the following to your `configuration.yaml` file:
+To add Met.no to your installation, go to Configuration >> Integrations in the UI and enable the Met.no integration.
+
+Confuration items:
```
Comment (klaasnicolaas) [inline source/_components/met.markdown]: Keep it between the {% configuration %} - {% endconfiguration %}

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by aidbish** [comment]
I think i have changed as requested. please bear with me, new at the github editing

**Comment by klaasnicolaas** [comment]
As you can see, this didn't work, there are now all new commits that have nothing to do with your PR. Rebase your own branch to that of the `current` on your fork and push it again to this PR.

**Comment by aidbish** [comment]
going to try again

**Pull request closed without merging**